ext_py/tests/test_fp_vector_slices.py

```
@@ -0,0 +1,293 @@
+    m = fp.Matrix.from_py(5, [[1, 1, 1], [2, 2, 2]])
+    m.row_mut(0).add(fp.FpVector.from_py(5, [3, 0, 1]), 1)
+    assert m.to_py() == [[4, 1, 2], [2, 2, 2]]
+
+
+def test_fp_slice_mut_new_method_errors_are_python_exceptions():
+    target_v = fp.FpVector(5, 3)
+    target = target_v.mut
+    same_prime_short = fp.FpVector(5, 2).const
+    other_prime = fp.FpVector(7, 3).const
+
+    with pytest.raises(ValueError):
```

Ruff (0.15.20)

[warning] `pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)